template/config/index.js

```diff
@@ -7,7 +7,19 @@ module.exports = {
     index: path.resolve(__dirname, '../dist/index.html'),
     assetsRoot: path.resolve(__dirname, '../dist'),
     assetsSubDirectory: 'static',
-    assetsPublicPath: '/',
+    assetsPublicPath: '/',{{#cdn}}
+    packages: [
+      {
+        name: 'vue',
+        var: 'Vue',
+        path: 'dist/vue.runtime.min.js'
+      }{{#router}},
```

Comment:
Can you make it dynamic instead? checking if vue-router and vuex are in the package.json file?